### DISCUSSION TITLE
Router 192.168.50.1 port unknown

### DISCUSSION THREAD

**Discussion by ShadowOffDeath** [rejetto/hfs#627] [General]
Verry strange after lates update i see this:
We cannot help you configuring your router because UPnP is not available.
Find more help [on this website](https://portforward.com/).
In old version cannot problems . My routhers is set 80 port forwarding long time ago.
![Port 80](https://github.com/rejetto/hfs/assets/168378955/e7ffdd69-c73d-4504-acfc-4f674838a607)

**Comment by rejetto** [comment]
"we cannot help" doesn't mean it's not working.
to know if it works you have to use the "verify" and see the result.
if the UPNP doesn't work, hfs doesn't know if the port is configured, but that's not a problem, you can proceed with the "verify".

often my modem's upnp is malfunctioning, and after i disable and reenable, it works again.

**Comment by rejetto** [comment]
always specify the versions you are referring to, don't just say "after update" and "old version"
